subsys/partition_manager/CMakeLists.txt

```
@@ -21,6 +21,20 @@ function(preprocess_pm_yml in_file out_file)
     message(FATAL_ERROR "command failed with return code: ${ret}")
   endif()
 
+  if (IMAGE_NAME)
```

Just a minor detail, that I don't believe have any practical effect here.
But I assume that it would be a little more correct to do:
```suggestion
  if (DEFINED IMAGE_NAME)
```
as that would be true, even if `IMAGE_NAME` is identical to the string `FALSE`, `IGNORE` etc.